docs/API/V1/endpoints.md

```diff
@@ -1,6 +1,263 @@
 # `GET /api/v1/endpoints`
 
 This API describes the set of URI endpoints available under the Pbench Server
-V1 API.
+V1 API, the Keycloak broker configuration for authorization, and the current
+Pbench Server version identification.
 
-__TBD__
+This API does not require authentication and has no access restrictions.
+
+## Response status
+
+`200`   **OK** \
+Successful request.
+
+## Response headers
+
+`content-type: application/json` \
+The return is a JSON document containing the summary "run" data from the
+dataset index.
+
+## Response body
+
+The `application/json` response body is a JSON object describing the Pbench
+Server configuration.
+
+The information is divided into four sections, as described below.
+
+### `api`
+
+A simplistic representation of the API endpoints supported on the Pbench Server.
+
+(This is mostly historical, as with the ongoing migration towards RESTful
+resource-oriented API paths, few of these API URIs can be used directly. Instead,
+see [URI](#uri) templates.)
+
+### `identification`
+
+This identifies the name and version of the Pbench Server.
+
+### `openid`
+
+The Pench Server authenticates through an OIDC broker (e.g., Keycloak). In order
+to authenticate and receive an authorization token to present to server APIs, the
+client must redirect to the broker login page using the `server_url` given here,
+with the Pbench Server `realm` and `client` ID.
+
+### `uri`
+
+A representation of the Pbench Server APIs supported on this server. Each is
+presented as a `name`, which is the key of the `uri` JSON object, containing
+`params` and `template` keys.
+
+#### `name`
```

Comment:
I don't see `name` in the sample result below:  under `"uri"` there are keys which are "convenience names" but none are `"name"`.

If you don't like `{name}`, then just take it out of monospace and capitalize it like a normal (sub-)header.

(As for the number of hashes, my eye doesn't really differentiate between three and four...so I don't know whether having five will actually help, but at least it'll be theoretically subordinate to four....)

Reply:
OK, I see now that you were probably reacting to the reference to "name" at line 49 above (which is the first one that the eye comes to), whereas I was referring to line 52 (and, since I selected only one line from the diff's, GitHub helpfully provided four for better context...).

Rather than "each is presented as a `name`", how changing the `uri` description to something like:
> A JSON object representing the available Pbench Server APIs:  each key corresponds to an API entry and maps to a sub-object containing entry's `params` and `template` descriptions.